modules/lib/lib-content/src/main/resources/lib/xp/content-graphql.js

```diff
@@ -0,0 +1,255 @@
+ *
+ * @returns {object} The current site as JSON.
+ */
+exports.getSite = function (params) {
+    var bean = __.newBean('com.enonic.xp.lib.content.GetSiteHandler');
+    bean.key = nullOrValue(params.key);
+    return __.toNativeObjectIncludeNullValues(bean.execute());
+};
+
+/**
+ * This function returns the site configuration for this app in the parent site of a content.
+ *
+ * @example-ref examples/content/getSiteConfig.js
+ *
+ * @param {object} params JSON with the parameters.
+ * @param {string} params.key Path or id to the content.
+ * @param {string} params.applicationKey Application key.
+ *
+ * @returns {object} The site configuration for current application as JSON.
+ */
+exports.getSiteConfig = function (params) {
+    var bean = __.newBean('com.enonic.xp.lib.content.GetSiteConfigHandler');
+    bean.key = nullOrValue(params.key);
+    bean.applicationKey = nullOrValue(params.applicationKey);
+    return __.toNativeObjectIncludeNullValues(bean.execute());
+};
+
+/**
+ * This function fetches children of a content.
+ *
+ * @example-ref examples/content/getChildren.js
+ *
+ * @param {object} params JSON with the parameters.
+ * @param {string} params.key Path or id to the parent content.
+ * @param {number} [params.start=0] Start index (used for paging).
+ * @param {number} [params.count=10] Number of contents to fetch.
+ * @param {string} [params.sort] Sorting expression.
+ *
+ * @returns {Object} Result (of content) fetched from the repository.
+ */
+exports.getChildren = function (params) {
+    var bean = __.newBean('com.enonic.xp.lib.content.GetChildContentHandler');
+    bean.key = required(params, 'key');
+    bean.start = params.start;
+    bean.count = params.count;
+    bean.sort = nullOrValue(params.sort);
+    return __.toNativeObjectIncludeNullValues(bean.execute());
+};
+
+/**
+ * This command queries content.
+ *
+ * @example-ref examples/content/query.js
+ *
+ * @param {object} params JSON with the parameters.
+ * @param {number} [params.start=0] Start index (used for paging).
+ * @param {number} [params.count=10] Number of contents to fetch.
+ * @param {string} params.query Query expression.
+ * @param {object} [params.filters] Filters to apply to query result
+ * @param {string} [params.sort] Sorting expression.
+ * @param {string} [params.aggregations] Aggregations expression.
+ * @param {string[]} [params.contentTypes] Content types to filter on.
+ *
+ * @returns {Object} Result of query.
+ */
+exports.query = function (params) {
+    var bean = __.newBean('com.enonic.xp.lib.content.QueryContentHandler');
+    bean.start = params.start;
+    bean.count = params.count;
+    bean.query = nullOrValue(params.query);
+    bean.sort = nullOrValue(params.sort);
+    bean.aggregations = __.toScriptValue(params.aggregations);
+    bean.contentTypes = __.toScriptValue(params.contentTypes);
+    bean.filters = __.toScriptValue(params.filters);
+    bean.highlight = __.toScriptValue(params.highlight);
```

Comment:
Codacy found an issue: ['__' is not defined.](https://app.codacy.com/manual/enonic/xp/pullRequest?prid=6266265)